### PR TITLE
wicked: Fix regression of file_replace_content()

### PR DESCRIPTION
**Pull request by cfconrad**
The commit 44085bad changed file_replace_content() that it escape '/'
characters of the value. Previously we escaped them manually,
which isn't necessary anymore.

- Verification run: 
  - http://cfconrad-vm.qa.suse.de/tests/4152 (advanced sut)
  - http://cfconrad-vm.qa.suse.de/tests/4154 (startandstop sut)

### CODE REVIEW

File: tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm [hunk 1]
```diff
@@ -21,12 +21,10 @@ use utils 'file_content_replace';
 
 sub run {
     my ($self, $ctx) = @_;
-    my $config   = '/etc/sysconfig/network/ifcfg-' . $ctx->iface() . '.42';
-    my $local_ip = $self->get_ip(type => 'vlan_changed', netmask => 1);
-    $local_ip =~ s'/'\\/';
+    my $config      = '/etc/sysconfig/network/ifcfg-' . $ctx->iface() . '.42';
     my $previous_ip = $self->get_ip(type => 'vlan', netmask => 1);
     $previous_ip =~ s'/'\\/';
```
Comment (cfconrad) [inline tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm]: I think we should escape `/` also in the regex? Should we  do it now or separate PR?
@asmorodskyi WDYT

Comment (asmorodskyi) [reply tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm]: instead of so matching escaping maybe we could just replace `/` with `|` ?

Comment (asmorodskyi) [reply tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm]: see for details https://netjunky.net/sed-replace-path-with-slash-separators/

Comment (cfconrad) [reply tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm]: yes we could do it, but then we need to escape `|`

Comment (cfconrad) [reply tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm]: follow up: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7379
